external/fv3fit/fv3fit/_shared/packer.py

```diff
@@ -39,7 +37,7 @@ def pack(data: xr.Dataset, sample_dim: str) -> Tuple[np.ndarray, pd.MultiIndex]:
     feature_dim_name = _unique_dim_name(data, sample_dim)
     stacked = data.to_stacked_array(feature_dim_name, sample_dims=[sample_dim])
     return (
-        stacked.transpose(sample_dim, feature_dim_name).data,
+        stacked.transpose(sample_dim, feature_dim_name).values,
```

Comment:
without this change, `pack` might return a `dask` array for example. We want `numpy.ndarray` in all cases.